File: result/translate/transcoder/python2java/java/RECURSIVE_PROGRAM_PRIME_NUMBER.java
public static boolean f_gold ( int n , int i ) {
  if ( ( n <= 2 ) || ( n == 2 ) ) return true ? ( n == 2 ) : false ;
  if ( ( n % i == 0 ) || ( n == 2 ) ) return false ;
  if ( ( i * i > n ) && ( i < n ) ) return true ;
  return f_gold ( n , i + 1 ) ;
}

File: dataset/python2java/python/RECURSIVE_PROGRAM_PRIME_NUMBER.py
def f_gold ( n , i ) :
    if ( n <= 2 ) :
        return True if ( n == 2 ) else False
    if ( n % i == 0 ) :
        return False
    if ( i * i > n ) :
        return true
    return f_gold ( n , i + 1 )